### PR TITLE
Display shopping list item names (issue 1)

### LINKED ISSUE #1: 1. As a user, I want to read the names of all of the items in my shopping list
## Summary

The most important feature of our app is the ability to look at what's on the list, so the user knows what they need to buy. The `List` component is the best place in our code to do that. In fact, this feature is already partially implemented. Refer to the acceptance criteria in this issue, and the existing code, to understand the work that remains.

The `List` component is rendered on the `/list` page. Because the navigation links are currently broken, you'll have to type that path into your browser's address bar.

## Acceptance criteria
- [x] The `streamListItems` function is used in `App.jsx` to get the items in the current list from the Firestore database
- [x] The list is passed into the `List` component as a prop named `data`
- [x] In `List.jsx`, the `ListItem` component is used to render the name of each item

## Notes
- ⚠️ Your **only goal** is to make sure that the names of the items render as expected. You might be tempted to modify `ListItem.jsx` to show more information – you will do that **later**. If you want a peek at the future, [this wireframe](https://github.com/the-collab-lab/smart-shopping-list/blob/main/_resources/wireframes/list-populated.png) is what your list could look like closer to the end of the project.
- The data you need is already being passed into the `List` component. What steps do you need to take in order to render it?

### PR DESCRIPTION
**Pull request by a-maffei**
## Description

In the `List` component, we now map through the shopping list items (received as `data` prop from `App`). For each item, we display a `ListItem` component: this component receives the name of the list item as a prop and renders it. 

## Related Issue

Closes #1 

## Acceptance Criteria

- [x] The `streamListItems` function is used in `App.jsx` to get the items in the current list from the Firestore database
- [x] The list is passed into the `List` component as a prop named `data`
- [x] In `List.jsx`, the `ListItem` component is used to render the name of each item

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Testing Steps / QA Criteria

* Visit the /list page
* Check if on the page, right under the title and the text "Hello from the /list page!" you can see a list of shopping list items

### CODE REVIEW

**Comment by a-maffei** [comment]
@naura1835 I pushed the code we worked on to our remote branch and created this PR draft. Is there anything you suggest to change in the PR before we mark it as "ready for review"?

**Comment by naura1835** [comment]
> @naura1835 I pushed the code we worked on to our remote branch and created this PR draft. Is there anything you suggest to change in the PR before we mark it as "ready for review"?

Everything is okay

**Review by iblas1 — COMMENTED** [review]
code checked, nice work. I think you are good to go.

**Comment by a-maffei** [comment]
Hey @Peacesandy, can you please take a look at our PR and branch code and let us know if you have any feedback? Thank you!

**Comment by Peacesandy** [comment]
Hi @a-maffei I've reviewed it, Awesome work

**Comment by a-maffei** [comment]
Hi @Amaka202, our PR has been approved by the rest of the team. Please let us know if you have any feedback for us, thank you!

File: src/views/List.jsx [hunk 1]
```diff
@@ -7,11 +8,11 @@ export function List({ data }) {
...
-				{/**
-				 * TODO: write some JavaScript that renders the `data` array
-				 * using the `ListItem` component that's imported at the top
-				 * of this file.
-				 */}
+				{data.map((item) => (
+					<Fragment key={item.id}>
+						<ListItem name={item.name} />
+					</Fragment>
+				))}
```
Comment (Amaka202) [inline src/views/List.jsx:15]: Good job team! I can also see you used the `key` to help react identify each item, that's a good coding practice!💥

**Review by Amaka202 — APPROVED** [review]
The Acs were met and you can go ahead to merge your branch✅✅

**Comment by a-maffei** [comment]
Thank you @Amaka202! Merging to main now :) cc @naura1835

**Pull request merged**